dora/tests/integration/src/test/java/alluxio/client/cli/fs/command/DoraLsCommandIntegrationTest.java

```diff
@@ -134,6 +136,9 @@ public void testLsR() throws Exception {
     );
   }
 
+  @DoraTestTodoItem(owner = "Hua", action = DoraTestTodoItem.Action.REMOVE,
+      comment = "currently access time is not recorded and updated")
+  @Ignore
   @Test
   public void testLsWithSortByAccessTime() throws IOException, AlluxioException {
```

Comment:
@huanghua78 I propose to remove this test as currently access time is not recorded and updated as part of the file metadata.

Reply:
It will also be very hard to implement access time tracking since the file is always free to be accessed out of band in the UFS.

Reply:
yeah there's a bug here and i encountered something similar

Reply:
I will fix this in https://github.com/TachyonNexus/enterprise/pull/4668

Reply:
thanks. For this PR, I will leave this test ignored, and you can un-ignore it when your fix lands.